addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -4,6 +4,25 @@ import options from 'web_editor.snippets.options';
 
 options.registry.WebsiteEvent.include({
 
+    /**
+     * @override
+     */
+    async start() {
+        const res = await this._super(...arguments);
+        // Only need for one RPC request as the option will be destroyed if a
+        // change is made.
+        const rpcData = await this._rpc({
+            model: 'event.event',
+            method: 'read',
+            args: [
+                [this.eventId],
+                ['meeting_room_allow_creation'],
+            ],
+        });
+        this.meetingRoomAllowCreation = rpcData[0]['meeting_room_allow_creation'];
```

Comment:
As it's to compute the state of the widget, this RPC probably needs to be performed each time the compute method uses it no?

If so it can be inlined in that method